src/__test__/backend/dataaccess/McodeV09SmartOnFhirDataSource.test.js

```diff
@@ -37,6 +37,11 @@ const sampleObservationSearchData = {
     entry: hardCodedFHIRPatient.entry.filter(e => e['resource']['resourceType'] === 'Observation')
 };
 
+const sampleReferenceResult = {
+    resourceType: 'Bundle',
+    entry: hardCodedFHIRPatient.entry.filter(e => e['resource']['resourceType'] === 'Condition')
+}
+
```

Comment:
[DP1] Since this is the expected response of a id-specific request, it should be the resource itself and not a bundle with the resource as an entry. For example, look at the response of `http://moonshot-dev.mitre.org:4002/baseDstu2/Patient/smart-1032702`

Reply:
You're definitely correct. I will update the format.